src/libs/OptionsListUtils.ts

```diff
@@ -2022,8 +2031,15 @@ function getOptions(
             }
 
             reportOption.isSelected = isReportSelected(reportOption, selectedOptions);
-
-            recentReportOptions.push(reportOption);
+            if (isCategorizeAction) {
+                const policyCategories = allPolicyCategories?.[`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${reportOption.policyID}`] ?? {};
+                const count = getEnabledCategoriesCount(policyCategories);
+                if (count !== 0) {
```

Comment:
NAB: We don't really need the count variable, it could be used in the condition directly.